mono/mini/mini-windows.c

```diff
@@ -152,6 +152,52 @@ get_win32_restore_stack (void)
 
 	return start;
 }
+#elif HAVE_API_SUPPORT_WIN32_RESET_STKOFLW && TARGET_ARM64
+static gpointer
+get_win32_restore_stack(void)
+{
+	static guint8* start = NULL;
+	guint8* code;
+
+	if (start)
+		return start;
+
+	const int size = 128;
+
+	/* restore_stack (void) */
+	start = code = mono_global_codeman_reserve(size);
+
+	arm64_push_reg(code, ARMREG_R5);
+	arm64_mov_reg_reg(code, ARMREG_R5, ARMREG_R4, 8);
+
+	/* push 32 bytes of stack space for Win64 calling convention */
+	arm64_alu_reg_imm(code, X86_SUB, ARMREG_R5, 32);
```

Comment:
I don't think this is needed. From https://devblogs.microsoft.com/oldnewthing/20220823-00/?p=107041:

> There is no parameter home space on the stack